cypress/integration/specs/nav.spec.ts

```diff
@@ -0,0 +1,163 @@
+import {NavbarPage} from '../../pages/navbarPage';
+
+describe('Navbar', () => {
+  before(() => {
+    NavbarPage.visitHome();
+  });
+
+  describe('Navbar Cluster Items', () => {
```

Comment:
This categorization is good, I'd only remove the `Navbar` from `Navbar Cluster Items` to not repeat it.